docs/content/uptobox.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Uptobox"
+description: "Rclone docs for Uptobox"
+---
+
+{{< icon "fa fa-archive" >}} Uptobox
+-----------------------------------------
+
+This is a Backend for Uptobox file storage service. Uptobox is closer to a one-click hoster than a traditional 
+cloud storage provider and therefore not suitable for long term storage. Please note that a Premium account is
+required to use this backend.
```

Comment:
This seemed to work with a free account for me?

Reply:
That's rather surprising to me. The docs state that you need to request a waiting token and wait up to 30 minutes between downloads. I'm completely skipping this step which supposedly only works when you have a premium account. Maybe that doesn't apply to files you uploaded yourself? I'll have to test that.